core/src/main/java/org/apache/gravitino/connector/AuthorizationPropertiesMeta.java

```diff
@@ -21,25 +21,166 @@
 import com.google.common.collect.ImmutableMap;
 import java.util.Map;
 
-public class AuthorizationPropertiesMeta {
+public class AuthorizationPropertiesMeta extends BasePropertiesMetadata
```

Comment:
It is recommended to use the `Metadata` suffix to maintain a consistent naming style.

Reply:
DONE.